frontend/src/routes/Governance/GovernancePage.tsx

```diff
@@ -36,6 +36,11 @@ export default function GovernancePage() {
               <AcmSecondaryNavItem isActive={!isOverview && location.pathname.startsWith(NavigationPath.policies)}>
                 <Link to={NavigationPath.policies}>{t('Policies')}</Link>
               </AcmSecondaryNavItem>
+              <AcmSecondaryNavItem
+                isActive={!isOverview && location.pathname.startsWith(NavigationPath.discoveredPolicies)}
```

Comment:
I think this can be simplified this
```suggestion
                isActive={location.pathname.startsWith(NavigationPath.discoveredPolicies)}
```